_data/forms/config-code.yml

```diff
@@ -5,68 +5,51 @@ id: code
 formGroups:
   - preset: schemaVersion
   - preset: adminCode
-  - preset: newAdmin
-    fields:
-      - preset: orgLevel
-      - widget: string
-        title: newAdminCode
-      - widget: select
-        title: ProvinceCode
-        options: [ab, bc, mb, nb, nl, nt, ns, nu, "on", pe, qc, sk, yt] # on is a yaml/liquid/jekyll keyword and thus must be escaped
-      - widget: string-i18n
-        title: Name
-        rule: alphanum-space
   - preset: selectCode
+  - preset: empty
+    start: true
+    title: releases
   - preset: description
   - preset: contact
   - preset: dates
     created: true
     modified: true
-    metadata: true
+    updated: true
   - widget: string-i18n
-    title: DownloadUrl
-    type: url
-    required: false
-  - widget: string-i18n
-    title: HomepageUrl
+    title: downloadUrl
     type: url
     required: false
+  - preset: homepageUrl
   - preset: languages
   - widget: string-i18n
-    title: Organization
+    title: organization
     rule: alphanum-space
     required: false
   - widget: group
-    title: Partner
+    title: partner
     required: false
     fields:
       - widget: string-i18n
-        title: Url
+        title: URL
```

Comment:
All title renamed without Capitalization and URL keyword spelled all caps. For consistency between schema and forms